specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/StaticSites.json

```diff
@@ -0,0 +1,1703 @@
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK."
+          },
+          "202": {
+            "description": "Asynchronous operation in progress."
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a static site": {
+            "$ref": "./examples/DeleteStaticSite.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "StaticSites"
+        ],
+        "summary": "Creates a new static site in an existing resource group, or updates an existing static site.",
+        "description": "Description for Creates a new static site in an existing resource group, or updates an existing static site.",
+        "operationId": "StaticSites_UpdateStaticSite",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the static site to create or update.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "staticSiteEnvelope",
+            "in": "body",
+            "description": "A JSON representation of the staticsite properties. See example.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/StaticSitePatchResource"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/StaticSiteARMResource"
+            }
+          },
+          "202": {
+            "description": "Asynchronous operation in progress.",
+            "schema": {
+              "$ref": "#/definitions/StaticSiteARMResource"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Patch a static site": {
+            "$ref": "./examples/PatchStaticSite.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/staticSites/{name}/authproviders/{authprovider}/users/list": {
```

Comment:
instead of users/list, you will have to make it listUSers to make it as an action over authProviders.

Reply:
or why cant it be  a collection GET on /users to list all the users. you already have a point GET on users

Reply:
We don't want any contributor to be able to do a GET and read these, to allow for RBAC we decided to let this be an action